collect_app/src/main/res/values/themes.xml

```diff
@@ -11,4 +11,10 @@
         <item name="android:buttonStyle">@style/Widget.Collect.Button.Custom</item>
         <item name="imageButtonStyle">@style/Widget.Collect.ImageButton.Custom.Light</item>
     </style>
+
+    <style name="Theme.Collect.Magenta" parent="Theme.Collect.BaseMagenta">
```

Comment:
I'd like to get away from having to use a base theme and a theme. Do we really need to have all image buttons and buttons take on custom styles? I'd rather just have the buttons in the map view explicitly add this style. 

It might seem nice to have things like this happen automatically but the trade-off is that it can be hard to work out why a view looks a certain way if you don't know a global customization is in place. Especially in a code base of this size!

Reply:
Do you mean for all the other themes, light, dark, and magenta, color for enabled buttons should be defined as a separate value in the theme.xml file, and then I can move away from `Theme.Collect.BaseLight` and `Theme.Collect.BaseDark` and have all the image buttons in map view use a style that defines custom colors for the enabled and disabled state like `@color/state_enabled` and like `@color/state_disabled`? 
This would define the style specifically for the image buttons in the map view. I am just skeptical about adding a new color resource in the themes file. Is this the best way to go about it?

Reply:
No I don't mean changing the light and dark theme setup. Magenta is an experiment to see how we can simplify theming and then we'll probably move to setting Dark/Light up in the same way after. 

These buttons are a a bit of weird setup as they should really be [Floating Action Buttons](https://material.io/components/buttons-floating-action-button) but I don't think we want to make that change now. For the moment I think we should give them a custom style (could be `MapButton`) and then define a tint list specific to that. We shouldn't need to use custom colors here either: I think we should follow what [Material Components does for the FAB](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/floatingactionbutton/res/color/mtrl_fab_bg_color_selector.xml) and use `colorSecondary` for enabled and `colorOnSurface` with a low alpha (so basically transparent) for disabled.

Reply:
Oh okay! That seems like a better solution. Thanks!